draft-ietf-ppm-dap.md

```diff
@@ -537,20 +537,19 @@ Task:
 
 # Overview {#overview}
 
-The protocol is executed by a large set of Clients and a pair of servers
-referred to as "Aggregators". Each Client's input to the protocol is its
-measurement (or set of measurements, e.g., counts of some user behavior). Given
-the input set of measurements `meas_1, ..., meas_N` held by the Clients, and an
-"aggregation parameter" `agg_param` shared by the Aggregators, the goal of DAP
-is to compute `agg_result = F(agg_param, meas_1, ..., meas_N)` for some
-function `F` while revealing nothing else about the measurements. We call `F`
-the "aggregation function" and `agg_result` the "aggregate result".
+The protocol is executed by a large set of Clients and a pair of Aggregators.
+Each Client's input to the protocol is its measurement (or set of measurements,
+e.g., counts of some user behavior). Given a set of measurements
+`meas_1, ..., meas_N` held by the Clients, and an "aggregation parameter"
+`agg_param` shared by the Aggregators, the goal of DAP is to compute
+`agg_result = F(agg_param, meas_1, ..., meas_N)` for some function `F` while
+revealing nothing else about the measurements. We call `F` the "aggregation
+function" and `agg_result` the "aggregate result".
 
 DAP is extensible in that it allows for the addition of new cryptographic
-schemes that compute different aggregation functions. In particular, the
-aggregation function is determined by the Verifiable Distributed Aggregation
-Function, or VDAF {{!VDAF=I-D.draft-irtf-cfrg-vdaf-14}}, used to securely
-compute it.
+schemes that compute different aggregation functions, determined by the
+Verifiable Distributed Aggregation Function, or
+{{!VDAF=I-D.draft-irtf-cfrg-vdaf-14}}, used to compute it.
```

Comment:
```suggestion
{{!VDAF=I-D.draft-irtf-cfrg-vdaf-14}}, used in the computation.
```
?

Reply:
I don't disagree with your phrasing but am not sure why it's better. Is the idea to allow for things besides the VDAF to be involved in the computation?

Reply:
I prefer the current verbiage: a VDAF computes an aggregation function, it's not just one part of the computation.